src/apic-extension/azext_apic_extension/aaz/latest/apic/integration/_create.py

```diff
@@ -11,14 +11,18 @@
 from azure.cli.core.aaz import *
 
 
+@register_command(
```

Comment:
When generating CLI code, please set this `create` command as `unregistered`. Otherwise, it will appear in the Azure CLI command list. We're using the `integration create amazon-api-gateway` and `integration create azure-api-management` to replace this create command for better UX experience.

Reply:
When I use aaz-dev to renerate the command, the register_command was automatically added. Should I remove it?

Reply:
I think we'd better to remove it.